const handleError = () => {
    throw new Error('[ERROR]');
};

export default handleError;
